### ISSUE TITLE
Feature Request: Not too verbose during model.fit

### ISSUE THREAD

**Issue by icyblade** [fchollet/keras#4005]
I wonder if it's possible that `model.fit` can print epochs once we have finished every 10% (of course this 10% can be a variable) of nb_epoches rather than every one single epoch.

For example, when we have `model.fit(x, y, verbose=2, nb_epochs=1000)`, the output will be something like:

> Epoch 1/1000
> 0s - loss: 0.0396 - acc: 0.0021
> Epoch 2/1000
> 0s - loss: 0.0239 - acc: 0.0024
> Epoch 3/1000
> 0s - loss: 0.0160 - acc: 0.0044
> Epoch 4/1000
> 0s - loss: 0.0130 - acc: 0.0042
> ... lots of staff here
> Epoch 1000/1000
> 0s - loss: 0.0020 - acc: 0.0042

This is what I want:

> Epoch 100/1000
> 0s - loss: 0.0396 - acc: 0.0021
> Epoch 200/1000
> 0s - loss: 0.0239 - acc: 0.0024
> Epoch 300/1000
> 0s - loss: 0.0160 - acc: 0.0044
> Epoch 400/1000
> 0s - loss: 0.0130 - acc: 0.0042
> Epoch 500/1000
> 0s - loss: 0.0020 - acc: 0.0042
> Epoch 600/1000
> 0s - loss: 0.0020 - acc: 0.0042
> Epoch 700/1000
> 0s - loss: 0.0020 - acc: 0.0042
> Epoch 800/1000
> 0s - loss: 0.0020 - acc: 0.0042
> Epoch 900/1000
> 0s - loss: 0.0020 - acc: 0.0042
> Epoch 1000/1000
> 0s - loss: 0.0020 - acc: 0.0042

**Comment by TBomberman** [comment]
I second this feature.

**Comment by nguyenqh** [comment]
It's surprising that this feature is not supported!

**Comment by TBomberman** [comment]
You can disable fit logging and pass in an epoch callback to print what you want when you want.
